app/src/ui/app.tsx

```diff
@@ -96,7 +96,7 @@ export class App extends React.Component<IAppProps, IAppState> {
       this.forceUpdate()
 
       requestIdleCallback(() => {
-        const now = Date.now()
+        const now = performance.now()
```

Comment:
Is there a reason why we're not using the same `process.hrtime` here? Is it not available in the renderer process?